src/components/Header/_mixins.scss

```diff
@@ -4,10 +4,22 @@
 /// @copyright IBM Security 2019
 ////
 
-@import '../../globals/index';
+@import '@carbon/layout/scss/mini-unit';
+@import '@carbon/layout/scss/spacing';
+@import '@carbon/themes/scss/mixins';
+@import '@carbon/type/scss/styles';
+@import 'carbon-components/scss/globals/scss/helper-mixins';
+@import 'carbon-components/scss/globals/scss/layout';
+
+@import '../../globals/motion/index';
+@import '../../globals/namespace/index';
 
 @import '../../platform/body/mixins';
 
+@import '../IconButton/mixins';
```

Comment:
This is one issue I couldn't really figure out - we'd talked about keeping imports from other components in the components `index.scss` but this doesn't always seem to work. Thought?

Reply:
In this file, we explicitly require the `IconButton` mixin, but not the styles themselves